guide/ru/fop.xconf

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<fop version="1.0">
+  <font-base>/home/david/GnucashDevelopment/Docs/gnucash-docs/./fonts</font-base>
```

Comment:
Hi Frank will do. I used an the XMLmind editor to edit the sources so that must have introduced the font. Strange that it into the document. Useless editorif it does that.

David

Reply:
Frank,

That has now beenremoved and the change pushed through to my repository. I don't think it was the XMLMind editor though as I did no editing of anything other thaHelp_ch_Transactions.xml.  I did initally build all the documentation and the build obviously picked it up from my computer. Presumable a make clean would have got rid of it as well.  I guess that is the problem with the in source build with autotools.
Cheers
David

Reply:
Yes, it is a typical artefact, if you build in-tree.

But now I am confused by your merge (3. commit). I will cherry-pick your clean 2. commit.
I will add a few more comments, but because of the merge, I will close this PR and ask you to improve the text/table in separate PRs.

Reply:
OK No problem Frank